### PR TITLE
Update Localization Doc

### PR DESCRIPTION
**Pull request by phucho0237**

### CODE REVIEW

File: docs/Localization.vi_VN.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+> Phiên bản của tài liệu này là v1. Nếu phiên bản chưa được cập nhật, vui lòng tham khảo [tài liệu gốc](./Localization.md).
```
Comment (Scighost) [inline docs/Localization.vi_VN.md:1]: v1 should be bolded

File: docs/Localization.vi_VN.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Sau khi dịch xong, vui lòng gửi lại về repo này thông qua Pull Request.
+
+
+## Dịch văn bản trong ứng dụng
+
+Bản dịch văn bản trong ứng dụng cho Starward được lưu trữ trên nền tảng [Crowdin](https://crowdin.com/project/starward), nơi bạn có thể sửa đổi nội dung văn bản bất cứ lúc nào. Nếu bạn muốn thêm ngôn ngữ dịch mới, vui lòng tạo [issue](https://github.com/Scighost/Starward/issues).
+
+Những thay đổi bạn thực hiện trên Crowdin sẽ được đồng bộ hóa với nhánh [l10n/main] [l10n/main](https://github.com/Scighost/Starward/tree/l10n/main) trong vòng 1 giờ đổng hồ, kích hoạt quá trình build tự động. Tìm quy trình làm việc mới nhất có tên `New Crowdin updates` trong [GitHub Actions](https://github.com/Scighost/Starward/actions/workflows/build.yml) và tải xuống tệp đã biên dịch (Artifacts). Bạn có thể kiểm tra hiệu ứng hiển thị của văn bản dịch trong ứng dụng theo thời gian thực. **Phiên bản đang được phát triển có thể làm hỏng cơ sở dữ liệu cá nhân của bạn `StarwardDatabase.db`. Vui lòng sao lưu trước khi kiểm tra. Phiên bản này không nên được sử dụng trong một thời gian dài.**
```
Comment (Scighost) [inline docs/Localization.vi_VN.md:37]: `[l10n/main]` appears twice

**Comment by Scighost** [comment]
By the way, I suggest renaming `Localization.vi_VN.md` to `Localization.vi-VN.md`

**Comment by phucho0237** [comment]
Done

File: docs/Localization.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-**v1** | English | [简体中文](./Localization.zh-CN.md)
+**v1** | English | [简体中文](./Localization.zh-CN.md) | [Tiếng Việt](./Localization.vi_VN.md)
```
Comment (Scighost) [inline docs/Localization.md:1]: You seem to have forgotten this place

**Comment by Scighost** [comment]
Thank you (｡･ω･｡)ﾉ♡

**Pull request merged**